<launch>
    <arg name="sim_time_required" default="false"/>
    <param name="use_sim_time" value="$(arg sim_time_required)"/>

    <arg name="semantic_map_path" default="$(find hydra)/config/kitti_360/colors.csv"/>
    <arg name="pointcloud_topic" value="/kitti360/cloud_labeled"/>

    <arg name="robot_frame" default="kitti360_gpsimu"/>
    <arg name="world_frame" default="map"/>
    <arg name="sensor_frame" default="kitti360_velodyne"/>

    <include file="$(find hydra_ros)/launch/hydra.launch" pass_all_args="true">
        <arg name="semantic_map_dir" value=""/>
        <arg name="semantic_map_file" value=""/>
        <arg name="robot_id" default="0"/>
        <arg name="use_gt_frame" value="true"/>
        <arg name="dsg_output_prefix" default="kitti_360"/>
        <arg name="config_dir" default="$(find hydra)/config/default_robot"/>
        <arg name="reconstruction_config" default="$(find hydra)/config/kitti_360/reconstruction_config.yaml"/>
        <arg name="extrinsics_mode" value="USE_TF"/>
        <arg name="rviz_dir" default="$(find hydra_ros)/rviz"/>
        <arg name="rviz_file" default="kitti_360.rviz"/>
        <arg name="use_oriented_bounding_boxes" default="true"/>
        <arg name="typology_dir" default="$(find hydra)/config/kitti_360"/>
        <arg name="typology_config" default="typology.yaml"/>
    </include>
</launch>
